packit/api.py

```diff
@@ -812,45 +815,10 @@ def clean(self):
         # command handlers have nothing to clean
         logger.debug("PackitAPI.cleanup (there are no objects to clean)")
 
-    def validate(self) -> str:
-        """ Create output for PackageConfig validation."""
-        schema_errors = PackageConfigValidation(self.package_config).validate()
-
-        if not schema_errors:
-            return (
-                f"{self.package_config.config_file_path} is valid and ready to be used"
-            )
-
-        output = f"{self.package_config.config_file_path} does not pass validation:\n"
-        for field_name, errors in schema_errors.items():
-            output += self.validate_get_field_output(errors, field_name)
-        return output
-
-    def validate_get_field_output(
-        self,
-        errors: Union[list, dict],
-        field_name: str,
-        field_category: str = "* field",
-        level: int = 1,
-    ) -> str:
-        if isinstance(errors, list):
-            field_output = f"{field_category} {field_name}: {errors[0]}\n"
-            return field_output
-
-        field_output = self.validate_get_field_item_output(errors, field_name, level)
-        return field_output
-
-    def validate_get_field_item_output(
-        self, errors: dict, field_name: str, level: int
-    ) -> str:
-        index_output = f"{level * '*'} field {field_name} has incorrect values:\n"
-        level += 1
-        for index, item_errors in errors.items():
-            index_type_error = self.validate_get_field_output(
-                item_errors,
-                index,
-                field_category=f"{level * '*'} value at index",
-                level=level,
-            )
-            index_output += index_type_error
-        return index_output
+    def validate_package_config(self, working_dir: Path) -> str:
```

Comment:
What about a static method, will this solve your problem with the required config?